addon/pods/date-range-picker/component.js

```diff
@@ -65,10 +35,103 @@ export default Component.extend(Picker, Clearable, PickerActions, EKMixin, Click
     return `${startDate}—${endDate}`;
   }),
 
+  selectedIndexAndCurrentPreset() {
+    let selectedIndex = this.selectedPresetIndex();
+    let currentPreset = this.get('presets')[selectedIndex];
+    if (currentPreset) {
+      Ember.set(currentPreset, "isSelected", false);
+    }
+    return [selectedIndex, currentPreset];
+  },
+
+  incrementSelectionIndex(currentPreset, currentIndex) {
+    if (currentIndex < 0 && !this.get('cancelSelected') && !this.get('applySelected')) { // just opened window
```

Comment:
This top conditional may be met by the else conditional, if there is no further collision in the elseif's